client/app/rules/page.tsx

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import React from 'react'
+import { Loading } from '@/_components/layout'
+import { Heading } from '@/_components/ui'
+import { useGetHouseQuery } from '@/_hooks/api'
+import { RuleList } from './component/RuleList'
+
+export default function Rules() {
+  const { data: house, isLoading } = useGetHouseQuery()
+
+  if (isLoading || !house) return <Loading />
```

Comment:
Maybe better to display Error message or component when failing fetching data rather than keep showing Loading component?

Reply:
There also might be a case where you successfully fetch data but you are not in any house.
In this case, you need to show a different message, but I will implement it in the futrue PR

Reply:
>Maybe better to display Error message or component when failing fetching data rather than keep showing Loading component?

I'm thinking to handle error in middleware if the ReactQuery does have such things. Or maybe axious. So for now, just leave the comment for error like this? `TODO: error handling`